examples/tts/conf/magpietts/magpietts_lhotse.yaml

```diff
@@ -87,6 +87,7 @@ model:
       shuffle: true
       num_workers: 6
       pin_memory: true
+      reweight_temperature: [1.0, 0.0]  # Temperature for re-weighting datasets. 1 is a neutral value. Lower temperature over-samples smaller datasets, and vice versa.
```

Comment:
Where is the documentation for what this parameter does? And can we put it in the same place as the other arguments?

Reply:
> * we should support this value being a float (same temperature for all levels) and also define what happens when this list is shorter than the number of nested groups in `input_cfg` (e.g., the last temperature is re-used for each next level).

i intentionally designed this with a strict `List` requirement to prioritize explicit configuration over flexibility. This ensures users are fully aware of their nested data structures and define temperatures intentionally.

but I agree that we need to validate that the number of nested groups matches the list length. I'll look into how to enforce that check.